H5P.BranchedVideo/branched-video.css

```diff
@@ -307,29 +308,22 @@ button:focus {
   color: white;
   margin : 0;
   position: absolute;
-  background-color: #b7b7b7;
+  background-color: #111111;
   padding-left: 2px;
   padding-right: 2px;
   padding-top: 0px;
 }
 
 @font-face {
    font-family: tapestry-font;
-   src: url('fonts/palanquin/palanquin-semibold.ttf');
+   src: url('fonts/arial/arial.ttf');
```

Comment:
If you're going to use Arial as the font, it's not necessary to include it as it is available on most machines already. Instead, just use the Arial font instead of tapestry-font in your CSS and use sans-serif as the alternate. i.e.: `font-family: Arial, sans-serif;` You can get rid of this @font-face declaration altogether and delete the fonts folder. Alternatively, if you want to keep this, I recommend using another font like https://fonts.google.com/specimen/Quicksand so it's worth the effort.